src/crawlers/cheerio_crawler.js

```diff
@@ -721,6 +721,7 @@ class CheerioCrawler extends BasicCrawler {
             timeoutSecs: this.requestTimeoutMillis / 1000,
             ...requestAsBrowserOptions,
             stream: true,
+            sessionToken: session,
```

Comment:
maybe we should move this above the `...requestAsBrowserOptions` line so it can be overridden via nav hooks